src/Models/Constants/Interfaces.ts

```diff
@@ -529,12 +530,10 @@ export interface IADTAdapter
     ): Promise<AdapterResult<ADTRelationshipsData>>;
 }
 
-export interface IJobsAdapter {
+export interface IJobsAdapter extends ADT3DSceneAdapter {
```

Comment:
hmmm Why do we need to extend the 3DScenes adapter? They shouldn't have much in common

Reply:
yes we do, because that's the adapter that environment picker wants